src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/TestComponent.scala

```diff
@@ -9,7 +9,7 @@ import slick.jdbc.JdbcProfile
 
 import scala.concurrent.ExecutionContext
 
-trait TestComponent extends FlatSpec with Matchers with ScalaFutures
+trait TestComponent extends Matchers with ScalaFutures
```

Comment:
don't define spec types in test base classes, it can mess things up for downstream users